enterprise/payara.common/nbproject/org-netbeans-modules-payara-common.sig

```diff
@@ -1490,48 +1490,11 @@ meth public org.netbeans.modules.payara.tooling.TaskState call2()
 supr org.netbeans.modules.payara.common.BasicTask<org.netbeans.modules.payara.tooling.TaskState>
 hfds LOGGER,RESTART_DELAY,support
 
-CLSS public final !enum org.netbeans.modules.payara.common.ServerDetails
```

Comment:
Changing `enum` to normal `class`? That is binary incompatible change. Now there are two options:
* do the change in a compatible way
* properly document/label/version an incompatible change

In case you want to evolve `enum` to non-enum in a compatible way, read [this essay](http://wiki.apidesign.org/wiki/Enum). It can be done, but just changing to `class ServerDetails` isn't the right approach.